examples/hotrod/create-image.sh

```diff
@@ -0,0 +1,5 @@
+cd $GOPATH/src/github.com/jaegertracing/jaeger
+make install
+cd $GOPATH/src/github.com/jaegertracing/jaeger/examples/hotrod
+CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o collector-linux main.go
+docker build -t jaegertracing/hotrod -f Dockerfile .
```

Comment:
I think this won't be enough, we need to add this to Travis build and publish to docker hub along with other images.

@pavolloffay how would you like to proceed here? This might be getting a bit more involved for @gbaufake who is new to this repo. We could merge this (without README changes) and then make the build changes.

Reply:
agree, We can merge this with dockerfile only. Then I can add publish and test for it. 

Rather then this script we could add make rules, otherwise there will be some repetition here and in the main makefile.